controllers/kubeadmconfig_controller.go

```diff
@@ -42,6 +42,7 @@ const (
 	// InfrastructureReadyAnnotationKey identifies when the infrastructure is ready for use such as joining new nodes.
 	// TODO move this into cluster-api to be imported by providers
 	InfrastructureReadyAnnotationKey = "cluster.x-k8s.io/infrastructure-ready"
```

Comment:
We should be able to remove this. In v1alpha2, we have `cluster.status.infrastructureReady` as a `bool`

Reply:
oh excellent